onboarding/include/amb_light_service.h

```diff
@@ -25,6 +25,11 @@ typedef enum {
  */
 uint8_t initLightService(void);
 
+/* Controller task config */
+#define CONTROLLER_NAME_LIGHT   "lightService"
```

Comment:
It looks like these are the config parameters for creating the light service task. These parameters are already defined above (check lines 12-14), so I'd just use those. 

For future reference, the names of these macros are a bit unclear. For example, `CONTROLLER_STACK_SIZE` sounds like it defines the stack size of the controller task which is already defined in `controller.h`. The macros above have more appropriate names